src/docker-tags.ts

```diff
@@ -1,4 +1,26 @@
-export const UPDATER_IMAGE_NAME =
-  'docker.pkg.github.com/dependabot/dependabot-updater:v1'
-export const PROXY_IMAGE_NAME =
-  'docker.pkg.github.com/github/dependabot-update-job-proxy:v1'
+import fs from 'fs'
+
+const dockerfile = fs.readFileSync(require.resolve('../Dockerfile'), 'utf8')
```

Comment:
This `require.resolve('../Dockerfile')` syntax is important, using the `path` module to resolve this path doesn't seem to work with `ncc build` and typescript, where it doesn't end up copying the `Dockerfile` to `dist/main`.

Reply:
We might want to add some guarantees that the built `dist/main/Dockerfile` is identical to the `Dockerfile` in root, maybe comparing the file contents in a ci check.